sample_files/pre-commit-13.0/.eslintrc

```diff
@@ -0,0 +1,289 @@
+{
+  "globals": {
+    "$": false,
+    "_": false,
+    "fuzzy": false,
+    "jQuery": false,
+    "moment": false,
+    "odoo": false,
+    "openerp": false,
+    "self": false
+  },
+  "env": {
+    "browser": true
+  },
+  "rules": {
+    "no-alert": "warn",
```

Comment:
We should have [some `ecmaVersion` setting](https://eslint.org/docs/user-guide/configuring#specifying-parser-options) here. It's important as it would be different depending on odoo versions.

Reply:
So far it's the same as in pylint-odoo with all errors as warnings. Feel free to propose improvements :)

Reply:
I used 2017 as discussed in https://github.com/OCA/pylint-odoo/pull/260#discussion_r332952195